jbrowse/circular/tracks.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2023 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+// this VCF has translocations for circular view
+export const circularTracks = [
+  {
+    type: "VariantTrack",
+    trackId: "pacbio_sv_vcf",
+    name: "HG002 Pacbio SV (VCF)",
+    assemblyNames: ["hg38"],
+    category: ["GIAB"],
+    adapter: {
+      type: "VcfTabixAdapter",
+      vcfGzLocation: {
+        uri: "http://localhost:3000/data/hs37d5.HG002-SequelII-CCS.bnd-only.sv.vcf.gz",
```

Comment:
Looks like the data for this is not available in this repository or publicly?

Can this data be added to the repo or hosted somewhere public for testing purposes?

Reply:
I'm hoping once this is in DMS-UI I can use our files in arranger, but we have to figure something out for the reference human genome